Makefile

```diff
@@ -161,39 +155,9 @@ fmt:
 	@$(GOFMT) -e -s -l -w $(ALL_SRC)
 	./scripts/updateLicenses.sh
 
-.PHONY: lint-gosec
-lint-gosec:
-	time gosec -quiet -exclude=G104,G107 ./...
-
-.PHONY: lint-staticcheck
-lint-staticcheck:
-	@cat /dev/null > $(LINT_LOG)
-	time staticcheck ./... \
-		| grep -v \
-			-e model/model.pb.go \
-			-e proto-gen \
-			-e _test.pb.go \
-			-e thrift-gen/ \
-			-e swagger-gen/ \
-		>> $(LINT_LOG) || true
-	@[ ! -s "$(LINT_LOG)" ] || (echo "Detected staticcheck failures:" | cat - $(LINT_LOG) && false)
-
 .PHONY: lint
-lint: lint-staticcheck lint-gosec
-	$(GOVET) ./...
-	$(MAKE) go-lint
-	@echo Running go fmt on ALL_SRC ...
-	@$(GOFMT) -e -s -l $(ALL_SRC) > $(FMT_LOG)
-	./scripts/updateLicenses.sh >> $(FMT_LOG)
-	./scripts/import-order-cleanup.sh stdout > $(IMPORT_LOG)
```

Comment:
I'm not sure goimport supports the grouping/ordering we want

Reply:
it supports grouping but it might create multiple groups.

Reply:
Actually, it's a good point, our script will enforce correct ordering of groups:

e.g.
```
import (
	"context"

	"github.com/jaegertracing/jaeger/cmd/collector/app/processor"
	"github.com/jaegertracing/jaeger/proto-gen/api_v2"

	"go.uber.org/zap"
	"google.golang.org/grpc/codes"
	"google.golang.org/grpc/status"
)
```
will give error:
```
cmd/collector/app/handler/grpc_handler.go imports out of order
```

but `goimports` gives no error. It does, however, enforce grouping of stdlib, 3rd party and local, just not ordering. I'll bring back our import checker script.